natural-language-understanding/custom_classifications_example.ipynb

```diff
@@ -187,310 +211,204 @@
     "    json.dump(nlu_data, f, indent=4)\n",
     "\n",
     "print('Data successfully converted to NLU format and saved locally in ' + training_data_filename)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "## Fetch data from NLC\n",
+    "## 3. How to Train a NLU Classifications Model\n",
```

Comment:
Are we removing the `Fetch data from NLC` section? We need this section actually in the notebook to be able to give customers an option to get their data from NLC.